src/pathfinding_service/api/rest.py

```diff
@@ -79,7 +79,7 @@ def _validate_args(args):
         return None
 
     # url parameters are used because json bodies for GET requests are uncommon
```

Comment:
does this comment make sense now that request is POST?

Reply:
You're right, it doesn't.